### DISCUSSION TITLE
Running tests on multiple isolated windows

### DISCUSSION THREAD

**Discussion by AkshayS967** [webdriverio/webdriverio#12950] [Q&A] [answered]
Is there any way to open an isolated browser window so that I can login into a window with user A and then login into another window as user B such that user A in the original window does not get logged out? I also want to switch control between the windows to test user interactions.

**Comment by christian-bromann** [comment, marked as answer]
For that you can use Multiremote which allows you to control two separate sessions: https://webdriver.io/docs/multiremote
